starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,62 @@
 // Names and Input
+console.log("I'm Ready!");
+var hacker1 = 'quino';
+var hacker2 = 'alberto';
+/*console.log("The driver's name is "+hacker1);
+message = "Second driver's name: ";
+var hacker2;
+hacker2 = window.prompt(message, "Please Type Here." );
+console.log("The navigator's name is"+ hacker2)
+*/
 
+for(var i = 0; i < hacker1.length; i++)
+{
+process.stdout.write(hacker1[i]+' ');
 
+}
+
+console.log(' ');
+
+for(var i = hacker1.length -1; i >= 0; i--)
+{
+process.stdout.write(hacker1[i]);
+}
 //Conditionals
+console.log("I'm Ready!");
+var hacker1 = 'alberto';
+var hacker2 = 'laura';
+
+/*console.log("The driver's name is "+hacker1);
+message = "Second driver's name: ";
+var hacker2;
+hacker2 = window.prompt(message, "Please Type Here." );
+console.log("The navigator's name is"+ hacker2)
+*/
+
+if(hacker1.length > hacker2.length)
+{
+  console.log('The Driver has the longest name, it has XX characters');
+
+}else if(hacker1.length < hacker2.length){
+ console.log('Yo, navigator got the longest name, it has XX characters');
+}else{
+ console.log('wow, you both got equally long names, XX characters!!');
+
```

Comment:
The only thing that you are missing is to display how much they are long. You should replace the XX in the console log with the value you want to print.